avm/utilities/pipelines/e2eValidation/resourceDeployment/New-TemplateDeployment.ps1

```diff
@@ -101,6 +101,9 @@ Optional. Maximum retry limit if the deployment fails. Default is 3.
 .PARAMETER doNotThrow
 Optional. Do not throw an exception if it failed. Still returns the error message though
 
+.PARAMETER RepoRoot
```

Comment:
This is the Inner function. Should we do the same for the main function `New-TemplateDeployment` (adding the input parameter with default value instead of retrieving it in the body)? At that point I'd pass the value from the pipeline, as the value would be a github variable and retrieving it would not be needed.

Reply:
done